templates/server/_nav_secondary.html

```diff
@@ -3,4 +3,5 @@
         <li{% if level_2 == 'hyperscale' %} class="last-item"{% endif %}><a{% if level_2 == 'management' %} class="active"{% endif %} href="/server/management">Server management</a></li>
         <li><a{% if level_2 == 'hyperscale' %} class="active"{% endif %} href="/server/hyperscale">Hyperscale</a></li>
         <li><a{% if level_2 == 'livepatch' %} class="active"{% endif %} href="/server/livepatch">Livepatch</a></li>
+        <li><a id="MAAS-menu-item" {% if level_2 == 'Server provisioning' %} class="active"{% endif %} href="/server/maas">Server provisioning</a></li>
```

Comment:
Should be `{% if level_2 == 'maas' %}` as its based on the URL structure not the title.